spacy/lang/punctuation.py

```diff
@@ -44,3 +44,23 @@
         r"(?<=[{a}0-9])[:<>=/](?=[{a}])".format(a=ALPHA),
     ]
 )
+
+
+# Some languages e.g. written with the Cyrillic alphabet permit the use of diacritics
+# to mark stressed syllables in words where stress is distinctive. Such languages
+# should use the COMBINING_DIACRITICS... suffix and infix regex lists in
+# place of the standard ones.
+COMBINING_DIACRITICS_TOKENIZER_SUFFIXES = list(TOKENIZER_SUFFIXES) + [
+    r"(?<=[{a}][{d}])\.".format(a=ALPHA, d=COMBINING_DIACRITICS),
```

Comment:
This regex includes single upper-case letters followed by a combining diacritic before a period. This is different from the behaviour for standard upper-case letters, where only two before a period are treated as a suffix. The difference is motivated by two things: the presence of the diacritic strongly suggests we are dealing with a real word rather than a random jumble of characters; and only handling two upper-case letters before a period taking diacritics into account would require several separate regexes because either or both could have a diacritic.